### ISSUE TITLE
Request: cabal-install 1.24 on Ubuntu 18.04 (Bionic)

### ISSUE THREAD

**Issue by asr** [haskell-CI/haskell-ci#399]
I didn't find `cabal-install-1.24` on Ubuntu 18.04 (Bionic) [here](https://launchpad.net/~hvr/+archive/ubuntu/ghc).

In Agda we are supporting various versions of GHC including 8.0.2. The  `cabal-install-1.24` program on Ubuntu 18.04 is needed for our CI infrastructure (see https://github.com/agda/agda/issues/4770).

**Comment by asr** [comment]
Any news?

**Comment by phadej** [comment]
I don't think it will happen. From Cabal maintainers perspective, there is hardly any reason to use old `cabal-install`s. You can (and *should*) use newer `cabal-install` with old(er) GHCs.

**Comment by asr** [comment]
Note the cabal-install 1.24 is [shipped](https://packages.ubuntu.com/search?keywords=cabal-install&searchon=names&suite=bionic&section=all) with Ubuntu 18.04. We do not want to require that Agda's users need to install a new version of cabal.

**Comment by phadej** [comment]
@asr can't you use that `cabal-install` from Ubuntu's official package repositories?

**Comment by asr** [comment]
Well, it could be possible but our [.travis.yml](https://github.com/agda/agda/blob/383e4920863216af5b9b8607056d7fb80322acab/.travis.yml) file is not trivial. I prefer don't write special instructions for GHC 8.0.2 and cabal-install 1.24 for keeping its maintenability.
